### PR TITLE
Add missing Ansible host groups for automated tess

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
Adds Ansible host groups 'dbservers' and 'monitoring'.

### CODE REVIEW

File: bin/knit-frog.py [hunk 1]
```diff
@@ -109,6 +109,10 @@
 cmdhandle.write('echo "$(hostname -f) ansible_connection=local ansible_user=$(whoami)"  >> ~/.hosts\n')
 cmdhandle.write("echo '[pulsarservers]' >> ~/.hosts\n")
 cmdhandle.write('echo "$(hostname -f) ansible_connection=local ansible_user=$(whoami)"  >> ~/.hosts\n')
+cmdhandle.write("echo '[dbservers]' >> ~/.hosts\n")
```
Comment (hexylena) [inline bin/knit-frog.py:112]: this should be `dbservers:children`, to match what is in ansible-galaxy, right?

Comment (sanjaysrikakulam) [reply bin/knit-frog.py:112]: I am sorry I am unable to associate which one you are comparing it to. This [file](https://github.com/hexylena/git-gat/blob/2023-04-05/galaxy.yml#L2) uses the hosts group `dbservers`.

Comment (hexylena) [reply bin/knit-frog.py:112]: https://github.com/hexylena/git-gat/blob/2023-04-05/hosts#L3

Comment (sanjaysrikakulam) [reply bin/knit-frog.py:112]: Sure! I will change it to `parent:children` and then the hostname line will be replaced by the `galaxyservers` group. I will push a commit.

**Review by hexylena — APPROVED** [review]

**Pull request merged**